buildpacks/nodejs-yarn/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- - Added Yarn version 4.0.0-rc.45.
```

Comment:
This line is duplicating the hyphen:
https://github.com/heroku/buildpacks-nodejs/blob/436b1e22e14c0318bb00f7ec0a58db41530ddeec/.github/workflows/inventory.yml#L25

Since it's already added here:
https://github.com/heroku/buildpacks-nodejs/blob/436b1e22e14c0318bb00f7ec0a58db41530ddeec/common/nodejs-utils/src/bin/diff_versions.rs#L54

Reply:
Thanks. See #551.